debezium-core/src/test/java/io/debezium/relational/history/KafkaDatabaseHistoryTest.java

```diff
@@ -29,11 +29,11 @@
 import io.debezium.doc.FixFor;
 import io.debezium.kafka.KafkaCluster;
 import io.debezium.relational.Tables;
-import io.debezium.relational.ddl.DdlParserSql2003;
-import io.debezium.relational.ddl.LegacyDdlParser;
 import io.debezium.text.ParsingException;
 import io.debezium.util.Collect;
 import io.debezium.util.Testing;
+import io.debezium.util.parser.DdlParserSql2003;
+import io.debezium.util.parser.LegacyDdlParser;
```

Comment:
How about moving the affected tests to the MySQL connector module then, so we could get rid of the Legacy parser impl altogether?

Reply:
@gunnarmorling Done. I am not sure if it is structurally clean but as a pragmatic solution it works.

Reply:
Yes, it's a bit of a blurry line. But maintaining a whole parser impl only for testing seems like too much of an overhead to me. I think it's the better approach now.